### ISSUE TITLE
travis testing needed.

### ISSUE THREAD

**Issue by daniel-wtd** [while-true-do/ansible-role-collabora-online#2]
### Type of Report

enhancement

### Actual behaviour

no testing provided/enabled

### Expected behaviour

Currently no testing is done. Testing should be enabled after #1 is done :)

**Comment by donvipre** [comment]
closing in favor of https://github.com/while-true-do/ansible-role-docker-collabora/issues/1
